app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -18,6 +18,9 @@ class ManageIQ::Providers::CloudManager::Vm < ::Vm
   has_many :floating_ips
   has_many :security_groups, :through => :network_ports
 
+  has_many :disks, :through => :hardware
```

Comment:
right, then all the :hardware => :disks and hardware.disks could go away, should I do that in the next PR?